build/scripts/create-tags

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# Important: This script should never cause the pipeline to fail, so that the tags can be created outside of it.
+# If run locally, it:
+# - does not need to run inside pmm-submodules as it clones all the necessary repos
+# - does not need to configure git to push using ssh, this user should set it all up ahead of time
+# - will use the user's creds and email to tag the repos, therefore sufficient permissions are required
+
+set +o errexit
+set +o nounset
+set -o xtrace
+
+# List of repositories whose release branches need to be tagged
+declare repos=(
+    "percona-platform/grafana"
+    "percona/grafana-dashboards"
+    "percona/pmm"
+    "Percona-Lab/pmm-submodules"
+)
+
+# These setting are only needed when running in CI (Jenkins or github actions)
+if [ -n "$CI" ]; then
+  # Configure git settings globally
+  git config --global advice.detachedHead false
+  git config --global user.email "noreply@percona.com"
+  git config --global user.name "PMM Jenkins"
+
+  # Configure git to push using ssh
+  export GIT_SSH_COMMAND="/usr/bin/ssh -i ${SSHKEY} -o StrictHostKeyChecking=no -o LogLevel=error -o UserKnownHostsFile=/dev/null"
+fi
+
+TAG="v${VERSION}"
+echo "We will be tagging repos with a tag: $TAG"
+
+for REPO in "${repos[@]}"; do
+    DIR_NAME=${REPO##*/} # grab the repo name from <company>/<repo>
+    rm -fr "$DIR_NAME" || true
+    mkdir -p "$DIR_NAME"
+
+    pushd "$DIR_NAME" >/dev/null
+        git clone https://github.com/${REPO} ./
+        # The default is https, and we want to set it to ssh
+        git remote set-url origin git@github.com:${REPO}.git
+
+        BRANCH="pmm-${VERSION}"
+        if ! git checkout "$BRANCH"; then
+            echo "Warning: failed to checkout the repository $REPO to $BRANCH"
+            popd >/dev/null
+            rm -rf "$DIR_NAME"
+            continue
+        fi
+        echo "SHA: $(git rev-parse HEAD)"
+
+        # If the tag already exists, we want to delete it and re-tag this SHA
+        if git tag -l "$TAG"; then
+            git tag --delete "$TAG"
+            git push --delete origin "$TAG"
+        fi
+
+        git tag --message="Version $TAG." "$TAG"
+        if ! git push origin "$TAG"; then
+            echo "Warning: failed to tag the repository $REPO with $TAG"
+        fi
```

Comment:
it shouldn't be just a warning, the script should fail

Reply:
> it shouldn't be just a warning, the script should fail

I believe it should not fail - it is part of the requirement as per the ticket.

Reply:
> Do tagging as the last step, so if it fails, we can still do it by hand. It should not break RPM pushing, Docker image tagging, etc.
it doesn't say that it shouldn't fail. It says that if it fails it shouldn't break RPM pushing, and it can be done simply by making it another Jenkins job which is called if the main one finished successfully. We already have this kind of job for devcontainer

Reply:
Pushing a tag to one repo could fail, the other three could succeed.

Reply:
Yes, and the whole job should fail if any of repos weren't tagged